projects/packages/videopress/src/client/admin/components/input/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { SearchIcon } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { useCallback, ChangeEvent, KeyboardEvent } from 'react';
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { InputProps, SearchInputProps } from './types';
+import type React from 'react';
+
+/**
+ * Input component
+ *
+ * @param {InputProps} props - Component props.
+ * @returns {React.ReactNode} - Input react component.
+ */
+export const Input: React.FC< InputProps > = ( {
```

Comment:
We should discourage the usage of `React.FC`.

Context:

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components
https://github.com/facebook/create-react-app/pull/8177